cmd/jaeger/internal/extension/jaegerquery/config.go

```diff
@@ -20,6 +20,7 @@ type Config struct {
 
 	TraceStoragePrimary     string `valid:"required" mapstructure:"trace_storage"`
 	TraceStorageArchive     string `valid:"optional" mapstructure:"trace_storage_archive"`
+	MetricsStorage          string `valid:"optional" mapstructure:"metrics_storage"`
```

Comment:
nit: let's stick to singular naming
```suggestion
	MetricStorage          string `valid:"optional" mapstructure:"metric_storage"`
```